+ source ../common/common_lib.sh
++ source /root/mugen-riscv/libs/locallibs/common_lib.sh
+++ python3 --version
+++ '[' 0 -eq 0 ']'
+++ source /root/mugen-riscv/libs/locallibs/common_lib_python.sh
+ main
++ type -t post_test
+ '[' -n function ']'
+ trap post_test EXIT INT HUP TERM
+ rpm -qa
+ grep expect
++ type -t config_params
+ '[' -n '' ']'
++ type -t pre_test
+ '[' -n function ']'
+ pre_test
+ LOG_INFO 'Start environmental preparation.'
+ message='Start environmental preparation.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Start environmental preparation.'
Thu Feb  2 23:32:14 2023 - INFO  - Start environmental preparation.
+ DNF_INSTALL open-iscsi
+ pkgs=open-iscsi
+ node=1
+ '[' -z '' ']'
+ tmpfile=
++ python3 /root/mugen-riscv/libs/locallibs/rpm_manage.py install --pkgs open-iscsi --node 1 --tempfile ''
+ tmpfile2=/tmp/tmp1uggv43d
+ '[' -z '' ']'
+ tmpfile=/tmp/tmp1uggv43d
+ LOG_INFO 'End of environmental preparation!'
+ message='End of environmental preparation!'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'End of environmental preparation!'
Thu Feb  2 23:32:53 2023 - INFO  - End of environmental preparation!
++ type -t run_test
+ '[' -n function ']'
+ run_test
+ LOG_INFO 'Start testing...'
+ message='Start testing...'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Start testing...'
Thu Feb  2 23:32:54 2023 - INFO  - Start testing...
+ test_execution iscsiuio.service
+ service=iscsiuio.service
++ date '+%Y-%m-%d %T'
+ log_time='2023-02-02 23:32:54'
+ test_restart iscsiuio.service
+ service=iscsiuio.service
+ systemctl restart iscsiuio.service
Failed to restart iscsiuio.service: Unit iscsiuio.service not found.
+ CHECK_RESULT 5 0 0 'iscsiuio.service restart failed'
+ actual_result=5
+ expect_result=0
+ mode=0
+ error_log='iscsiuio.service restart failed'
+ '[' -z 5 ']'
+ '[' 0 -eq 0 ']'
+ test 5x '!=' 0x
+ test -n 'iscsiuio.service restart failed'
+ LOG_ERROR 'iscsiuio.service restart failed'
+ message='iscsiuio.service restart failed'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'iscsiuio.service restart failed'
Thu Feb  2 23:32:54 2023 - ERROR - iscsiuio.service restart failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 34'
+ message='../common/common_lib.sh line 34'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 34'
Thu Feb  2 23:32:55 2023 - ERROR - ../common/common_lib.sh line 34
+ return 0
+ SLEEP_WAIT 5
+ wait_time=5
+ cmd=
+ mode=1
+ python3 /root/mugen-riscv/libs/locallibs/sleep_wait.py --time 5 --cmd '' --mode 1
+ systemctl status iscsiuio.service
+ grep 'Active: active'
Unit iscsiuio.service could not be found.
+ CHECK_RESULT 1 0 0 'iscsiuio.service restart failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='iscsiuio.service restart failed'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'iscsiuio.service restart failed'
+ LOG_ERROR 'iscsiuio.service restart failed'
+ message='iscsiuio.service restart failed'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'iscsiuio.service restart failed'
Thu Feb  2 23:33:01 2023 - ERROR - iscsiuio.service restart failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 37'
+ message='../common/common_lib.sh line 37'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 37'
Thu Feb  2 23:33:02 2023 - ERROR - ../common/common_lib.sh line 37
+ return 0
+ systemctl stop iscsiuio.service
Failed to stop iscsiuio.service: Unit iscsiuio.service not loaded.
+ CHECK_RESULT 5 0 0 'iscsiuio.service stop failed'
+ actual_result=5
+ expect_result=0
+ mode=0
+ error_log='iscsiuio.service stop failed'
+ '[' -z 5 ']'
+ '[' 0 -eq 0 ']'
+ test 5x '!=' 0x
+ test -n 'iscsiuio.service stop failed'
+ LOG_ERROR 'iscsiuio.service stop failed'
+ message='iscsiuio.service stop failed'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'iscsiuio.service stop failed'
Thu Feb  2 23:33:03 2023 - ERROR - iscsiuio.service stop failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 39'
+ message='../common/common_lib.sh line 39'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 39'
Thu Feb  2 23:33:04 2023 - ERROR - ../common/common_lib.sh line 39
+ return 0
+ SLEEP_WAIT 5
+ wait_time=5
+ cmd=
+ mode=1
+ python3 /root/mugen-riscv/libs/locallibs/sleep_wait.py --time 5 --cmd '' --mode 1
+ systemctl status iscsiuio.service
+ grep 'Active: inactive'
Unit iscsiuio.service could not be found.
+ CHECK_RESULT 1 0 0 'iscsiuio.service stop failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='iscsiuio.service stop failed'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'iscsiuio.service stop failed'
+ LOG_ERROR 'iscsiuio.service stop failed'
+ message='iscsiuio.service stop failed'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'iscsiuio.service stop failed'
Thu Feb  2 23:33:10 2023 - ERROR - iscsiuio.service stop failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 42'
+ message='../common/common_lib.sh line 42'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 42'
Thu Feb  2 23:33:11 2023 - ERROR - ../common/common_lib.sh line 42
+ return 0
+ systemctl start iscsiuio.service
Failed to start iscsiuio.service: Unit iscsiuio.service not found.
+ CHECK_RESULT 5 0 0 'iscsiuio.service start failed'
+ actual_result=5
+ expect_result=0
+ mode=0
+ error_log='iscsiuio.service start failed'
+ '[' -z 5 ']'
+ '[' 0 -eq 0 ']'
+ test 5x '!=' 0x
+ test -n 'iscsiuio.service start failed'
+ LOG_ERROR 'iscsiuio.service start failed'
+ message='iscsiuio.service start failed'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'iscsiuio.service start failed'
Thu Feb  2 23:33:12 2023 - ERROR - iscsiuio.service start failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 44'
+ message='../common/common_lib.sh line 44'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 44'
Thu Feb  2 23:33:12 2023 - ERROR - ../common/common_lib.sh line 44
+ return 0
+ SLEEP_WAIT 5
+ wait_time=5
+ cmd=
+ mode=1
+ python3 /root/mugen-riscv/libs/locallibs/sleep_wait.py --time 5 --cmd '' --mode 1
+ systemctl status iscsiuio.service
+ grep 'Active: active'
Unit iscsiuio.service could not be found.
+ CHECK_RESULT 1 0 0 'iscsiuio.service start failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='iscsiuio.service start failed'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'iscsiuio.service start failed'
+ LOG_ERROR 'iscsiuio.service start failed'
+ message='iscsiuio.service start failed'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'iscsiuio.service start failed'
Thu Feb  2 23:33:19 2023 - ERROR - iscsiuio.service start failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 47'
+ message='../common/common_lib.sh line 47'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 47'
Thu Feb  2 23:33:19 2023 - ERROR - ../common/common_lib.sh line 47
+ return 0
+ test_enabled iscsiuio.service
+ service=iscsiuio.service
++ systemctl is-enabled iscsiuio.service
Failed to get unit file state for iscsiuio.service: No such file or directory
+ state=
+ '[' '' == enabled ']'
+ '[' '' == disabled ']'
+ '[' '' == masked ']'
+ '[' '' == static ']'
+ LOG_INFO 'Unit is indirect, ignoring.'
+ message='Unit is indirect, ignoring.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Unit is indirect, ignoring.'
Thu Feb  2 23:33:20 2023 - INFO  - Unit is indirect, ignoring.
+ journalctl --since '2023-02-02 23:32:54' -u iscsiuio.service
+ grep -i 'fail\|error'
+ grep -v -i 'DEBUG\|INFO\|WARNING'
+ CHECK_RESULT 1 0 1 'There is an error message for the log of iscsiuio.service'
+ actual_result=1
+ expect_result=0
+ mode=1
+ error_log='There is an error message for the log of iscsiuio.service'
+ '[' -z 1 ']'
+ '[' 1 -eq 0 ']'
+ test 1x == 0x
+ return 0
+ test_reload iscsiuio.service
+ service=iscsiuio.service
+ systemctl start iscsiuio.service
Failed to start iscsiuio.service: Unit iscsiuio.service not found.
+ systemctl reload iscsiuio.service
+ grep 'Job type reload is not applicable'
+ CHECK_RESULT 1 0 0 'Job type reload is not applicable for unit iscsiuio.service'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='Job type reload is not applicable for unit iscsiuio.service'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'Job type reload is not applicable for unit iscsiuio.service'
+ LOG_ERROR 'Job type reload is not applicable for unit iscsiuio.service'
+ message='Job type reload is not applicable for unit iscsiuio.service'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Job type reload is not applicable for unit iscsiuio.service'
Thu Feb  2 23:33:21 2023 - ERROR - Job type reload is not applicable for unit iscsiuio.service
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 91'
+ message='../common/common_lib.sh line 91'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 91'
Thu Feb  2 23:33:22 2023 - ERROR - ../common/common_lib.sh line 91
+ return 0
+ systemctl status iscsiuio.service
+ grep 'Active: active'
Unit iscsiuio.service could not be found.
+ systemctl status iscsiuio.service
+ grep 'inactive (dead)'
Unit iscsiuio.service could not be found.
+ return 1
+ LOG_INFO 'Finish test!'
+ message='Finish test!'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Finish test!'
Thu Feb  2 23:33:22 2023 - INFO  - Finish test!
+ CASE_RESULT 0
+ case_re=0
+ test -z 7
+ test 7 -gt 0
+ LOG_ERROR 'failed to execute the case.'
+ message='failed to execute the case.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'failed to execute the case.'
Thu Feb  2 23:33:23 2023 - ERROR - failed to execute the case.
+ exit 7
+ post_test
+ LOG_INFO 'start environment cleanup.'
+ message='start environment cleanup.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'start environment cleanup.'
Thu Feb  2 23:33:24 2023 - INFO  - start environment cleanup.
+ systemctl stop iscsiuio.service
Failed to stop iscsiuio.service: Unit iscsiuio.service not loaded.
+ systemctl stop iscsid.service
+ DNF_REMOVE
+ node=1
+ pkg_list=
+ mode=0
+ [[ -z /tmp/tmp1uggv43d ]]
+ '[' 0 -ne 0 ']'
+ '[' 1 == 0 ']'
+ python3 /root/mugen-riscv/libs/locallibs/rpm_manage.py remove --node 1 --pkgs '' --tempfile /tmp/tmp1uggv43d
+ '[' 0 -ne 0 ']'
+ LOG_INFO 'Finish environment cleanup!'
+ message='Finish environment cleanup!'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Finish environment cleanup!'
Thu Feb  2 23:33:40 2023 - INFO  - Finish environment cleanup!
